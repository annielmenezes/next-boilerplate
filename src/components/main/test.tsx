import { render, screen } from '@testing-library/react'

import Main from '.'

describe('Component: <Main />', () => {
  it('shold render the heading', () => {
    render(<Main />)

    expect(
      screen.getByRole('heading', { name: /NextJS Boilerplate/i })
    ).toBeInTheDocument()
  })
})
