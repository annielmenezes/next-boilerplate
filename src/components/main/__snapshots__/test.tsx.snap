// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: <Main /> shold render the heading 1`] = `
.c0 {
  width: 100%;
  height: 100%;
  background-color: #f5f5f5;
  color: #222;
  text-align: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c1 {
  font-size: 2.5rem;
}

.c2 {
  font-size: 2rem;
}

<main
  class="c0"
>
  <h1
    class="c1"
  >
    NextJS Boilerplate
  </h1>
  <h2
    class="c2"
  >
    Typescript, ReactJS, NextJS and Styled Components
  </h2>
</main>
`
